rollup.config.js

```diff
@@ -3,16 +3,15 @@ const rollup = require('rollup');
 // plugin that transpiles output into commonjs format
 const commonjs = require('@rollup/plugin-commonjs');
 // plugin that transpiles es6 to es5 for legacy platforms
-const buble = require('@rollup/plugin-buble');
```

Comment:
The code output is cleaner without Buble, and all browser nowadays support all the syntax being used (I've been using these ES module output at NASA with no issues). This eliminated errors Buble was having on newer syntax that I didn't want to spend time configuring (by wasting time adding plugins to acorn) when all browsers already support the syntax out of the box.